src/proto/gps.proto

```diff
@@ -20,6 +20,9 @@ enum SampleType {
   TOTAL_DATA_TYPES = 13;
   POSITION = 14;
   LINEAR_VELOCITY = 15;
+  RGB_IMAGE_SIZE = 16;
+  DOMAIN_IMAGE = 17;
```

Comment:
I see. So for example if you have a maze and you want to show the agent the maze at the beginning of the trial? 

@cbfinn can you think of a concise way to word this for the comment?